.github/workflows/manubot.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
           restore-keys: |
             ci-cache-refs/heads/master
       - name: Install Environment
-        uses: goanpeca/setup-miniconda@v1
+        uses: conda-incubator/setup-miniconda@v1
```

Comment:
Regarding the warnings at https://github.com/manubot/rootstock/runs/1364210024#step:5:178, we'll probably have to upgrade to `conda-incubator/setup-miniconda@v2` when that is released as per https://github.com/conda-incubator/setup-miniconda/issues/84